### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by Mahyar1982**
Login Activity is ready now!

### CODE REVIEW

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        android:id="@+id/textLog"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Are you new?please Register"
+        android:layout_marginTop="18dp"
+        android:layout_below="@+id/buttonLog"
+        android:layout_centerHorizontal="true" />
+
+    <EditText
+        android:id="@+id/nameReg"
```
Comment (m3hrad) [inline app/src/main/res/layout/activity_login.xml]: Let's have first name and last name separately.

File: app/src/test/java/com/bros/freetimelogin/ExampleUnitTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.bros.freetime;
+package com.bros.freetimelogin;
```
Comment (m3hrad) [inline app/src/test/java/com/bros/freetimelogin/ExampleUnitTest.java]: Should be freetime

File: readme.MD [hunk 3]
```diff
@@ -1,2 +0,0 @@
-Second change in Readme.
```
Comment (m3hrad) [inline readme.MD:1]: This file shouldn't be removed.

**Comment by Mahyar1982** [comment]
The package name changed to "freetime"

**Review by m3hrad — APPROVED** [review]
Good job 👍

**Pull request merged**